src/Styleguide/Elements/RulerSeparator.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps, themeGet } from "styled-system"
+
+const HorizontalRuler = styled.div.attrs<SpaceProps>({})`
```

Comment:
I figured this might be useful in other place on the page. I think it can be way more configurable. But for now just creating this placeholder to use for a sidebar

Reply:
Maybe just `<HR />`?